### ISSUE TITLE
[(zoom)] two-way binding only works with numeric values

### ISSUE THREAD

**Issue by Xleg** [stephanrauh/ngx-extended-pdf-viewer#556]
Hello,
I had an issue with the [(zoom)] two-way binding attribute. If I switch the zoom from an numeric value to a string value then my zoom variable does not change. I noticed that it's a normal behaviour that the (zoomChange) event is only triggerd by numeric values. So my question is, is it also a wanted behavior for this attribute?
Nethertheless it would be great if the variable is also updated when the zoom value is set to a string, because currently I can't use the zoom accross multiple PDFs when it's a string value. It's only working with percentages.

**Example to reproduce**

My HTML file:
```html
<ngx-extended-pdf-viewer *ngIf="pdfObj" [base64Src]="pdfObj" [textLayer]="true" [(zoom)]="pdfZoom"
  useBrowserLocale="true" height="100vh" [showFindButton]="true">
</ngx-extended-pdf-viewer>
```
In my .ts file I define `pdfZoom`:
```typescript
pdfZoom: number | string | undefined = 'page-fit';
```
Now if I switch my zoom from `page-fit` to `50%` and then switch to a new pdf document it's working. The zoom of the new pdf document is also `50%`. But if I switch from `page-fit` to `auto` and change my document it's not passing the value accross. It's reset to `page-fit`.

**Version info**
ngx-extended-pdf-viewer@6.1.0

**Desktop (please complete the following information):**
 - Browser: Google Chrome Version 86.0.4240.193 (Offical Build) (64-Bit)

Thank you!

**Comment by stephanrauh** [comment]
@xycheung This demo matches your description: https://pdfviewer.net/zoom

However, I can't reproduce the bug over the. What are you doing differently?

**Comment by Xleg** [comment]
Hello,
thanks for your answer.
I know that demo, but in the demo I can't change the document source.

So heres a another example to reproduce:

In html:
```html
<h1>Test</h1>
<div class="form-group">
  <label for="exampleFormControlSelect1">Select a dummy pdf</label>
  <select class="form-control" id="exampleFormControlSelect1" [(ngModel)]="source">
    <option [value]="firstPDF">1st dummy pdf</option>
    <option [value]="secondPDF">2nd dummy pdf</option>
  </select>
</div>

<div>
  <p>Current zoom setting: {{pdfZoom}}</p>
</div>

<ngx-extended-pdf-viewer *ngIf="source" [src]="source" [textLayer]="true" [(zoom)]="pdfZoom" theme="dark"
  useBrowserLocale="true" height="100vh" [showFindButton]="true">
</ngx-extended-pdf-viewer>
```
In component.ts:
```typescript
pdfZoom: number | string | undefined = 'page-fit';
source;
firstPDF = 'https://www.w3.org/WAI/ER/tests/xhtml/testfiles/resources/pdf/dummy.pdf';
secondPDF = 'https://raw.githubusercontent.com/mozilla/pdf.js/ba2edeae/examples/learning/helloworld.pdf';

constructor() {
    this.source = this.firstPDF;
  }
```

**To reproduce:**
1. Select a pdf file and set the zoom from `page-fit` to a numeric value (e.g. 100%)
The zoom value will switch to `100`.
2. Switch to the other pdf file.
Everything works fine and the new file is loaded with a zoom of 100%.
3. Repeat the process with a string value.
For example: Set the zoom from `page-fit` to `auto`.
The zoom value will not switch. But the zoom in the viewer changes.
4. If you load the other pdf now, the zoom is still `page-fit`.

**Comment by stephanrauh** [comment]
I've implemented the reproducer here: https://github.com/stephanrauh/ngx-extended-pdf-viewer-issues/tree/master/issue556-non-numeric-zoom

**Comment by stephanrauh** [comment]
Your bugfix has landed with version 7.0.0-beta.0. Fixing the behavior of `[(zoom)]` is probably breaking applications relying on the bug, hence the new major version.

**Comment by Xleg** [comment]
It works. Thank you!
